Makefile

```diff
@@ -68,70 +92,92 @@ vet:
 
 push-controller-image: build-controller-image
-	$(CONTAINER_CMD) tag cmd/forklift-controller:forklift-controller-image ${CONTROLLER_IMAGE}
-	$(CONTAINER_CMD) push ${CONTROLLER_IMAGE}
+	$(CONTAINER_CMD) tag bazel/cmd/forklift-controller:forklift-controller-image $(CONTROLLER_IMAGE)
+	$(CONTAINER_CMD) push $(CONTROLLER_IMAGE)
 
 build-api-image:
-	bazel run cmd/forklift-api:forklift-api-image
+	bazel run cmd/forklift-api:forklift-api-image \
+		--sandbox_writable_path=$${HOME}/.ccache \
+		--sandbox_writable_path=$${XDG_RUNTIME_DIR}/ccache-tmp
 
 push-api-image: build-api-image
-	$(CONTAINER_CMD) tag cmd/forklift-api:forklift-api-image ${API_IMAGE}
-	$(CONTAINER_CMD) push ${API_IMAGE}
+	$(CONTAINER_CMD) tag bazel/cmd/forklift-api:forklift-api-image $(API_IMAGE)
+	$(CONTAINER_CMD) push $(API_IMAGE)
 
 build-validation-image:
 	bazel run validation:forklift-validation-image
 
 push-validation-image: build-validation-image
-	$(CONTAINER_CMD) tag validation:forklift-validation-image ${VALIDATION_IMAGE}
-	$(CONTAINER_CMD) push ${VALIDATION_IMAGE}
+	$(CONTAINER_CMD) tag bazel/validation:forklift-validation-image $(VALIDATION_IMAGE)
+	$(CONTAINER_CMD) push $(VALIDATION_IMAGE)
 
 build-operator-image:
 	bazel run operator:forklift-operator-image
 
 push-operator-image: build-operator-image
-	$(CONTAINER_CMD) tag operator:forklift-operator-image ${OPERATOR_IMAGE}
-	$(CONTAINER_CMD) push ${OPERATOR_IMAGE}
+	$(CONTAINER_CMD) tag bazel/operator:forklift-operator-image $(OPERATOR_IMAGE)
+	$(CONTAINER_CMD) push $(OPERATOR_IMAGE)
 
 build-virt-v2v-image:
 	bazel run virt-v2v:forklift-virt-v2v
 
 push-virt-v2v-image: build-virt-v2v-image
-	$(CONTAINER_CMD) tag virt-v2v:forklift-virt-v2v ${VIRT_V2V_IMAGE}
-	$(CONTAINER_CMD) push ${VIRT_V2V_IMAGE}
+	$(CONTAINER_CMD) tag bazel/virt-v2v:virt-v2v-image $(VIRT_V2V_IMAGE)
+	$(CONTAINER_CMD) push $(VIRT_V2V_IMAGE)
 
 build-operator-bundle-image:
-	bazel run operator:forklift-operator-bundle-image --action_env CONTROLLER_IMAGE=${CONTROLLER_IMAGE} --action_env VALIDATION_IMAGE=${VALIDATION_IMAGE} --action_env OPERATOR_IMAGE=${OPERATOR_IMAGE} --action_env VIRT_V2V_IMAGE=${VIRT_V2V_IMAGE} --action_env API_IMAGE=${API_IMAGE}
+	bazel run operator:forklift-operator-bundle-image \
+		--action_env VERSION=$(VERSION) \
+		--action_env NAMESPACE=$(NAMESPACE) \
+		--action_env CHANNELS=$(CHANNELS) \
+		--action_env DEFAULT_CHANNEL=$(DEFAULT_CHANNEL) \
+		--action_env OPERATOR_IMAGE=$(OPERATOR_IMAGE) \
+		--action_env MUST_GATHER_IMAGE=$(MUST_GATHER_IMAGE) \
+		--action_env MUST_GATHER_API_IMAGE=$(MUST_GATHER_API_IMAGE) \
+		--action_env UI_IMAGE=$(UI_IMAGE) \
+		--action_env UI_PLUGIN_IMAGE=$(UI_PLUGIN_IMAGE) \
+		--action_env VALIDATION_IMAGE=$(VALIDATION_IMAGE) \
+		--action_env VIRT_V2V_IMAGE=$(VIRT_V2V_IMAGE) \
+		--action_env CONTROLLER_IMAGE=$(CONTROLLER_IMAGE) \
+		--action_env API_IMAGE=$(API_IMAGE) \
+		--action_env VIRT_V2V_DONT_REQUEST_KVM=$(VIRT_V2V_DONT_REQUEST_KVM)
```

Comment:
does it mean that it would be added to ForkliftController or why do we pass it here? we didn't want this option to be so visible

Reply:
Yes, the idea was to have everything customizable from the Makefile so we don't need to run any `bazel` command by hand, but I can delete this var if you don't want it to be visible within the Makefile.

Reply:
Removed from the Makefile.